include/game.h

```diff
@@ -0,0 +1,20 @@
+#ifndef GAME_H
+#define GAME_H
+
+#include "screen.h"
+
+class Game {
+private:
+  // Private Variables
+  Screen* m_screen;
```

Comment:
If this class is supposed to "own" the `Screen`, you should use a `std::unique_ptr`. For general pointer safety, everything on the heap should be owned by something. This is done by using a `std::unique_ptr`.